gitops/azure-devops/build.sh

```diff
@@ -0,0 +1,186 @@
+function init() {
+    cp -r * $HOME/
+    cd $HOME
+
+    echo "CHECKING GIT HOST"
+    if [[ "$GIT_HOST" == "github" ]]; then
+        git_dest_repo="https://github.com/$AKS_MANIFEST_REPO"
+        git_type=$GIT_HOST
+    elif [[ "$GIT_HOST" == "azure" ]]; then
+        git_dest_repo="https://dev.azure.com/$AKS_MANIFEST_REPO" # For repos that reside in Azure Devops, the AKS_MANIFEST_REPO should be formatted like "user_account/project_name/_git/repo_name"
+        git_type="dev.azure"
+    else
+        echo 'Git host not specified in variable $GIT_HOST'
+        exit 1
+    fi
+
+    echo "VERIFYING PERSONAL ACCESS TOKEN"
+    if [ -n "$ACCESS_TOKEN" ]
+    then
+        echo "Personal Access token defined for git host: $GIT_HOST"
+    else
+        exit 1
+    fi
+}
+
+# Initialize Helm
+function helm_init() {
```

Comment:
Could someone reach out to me about the function of this - it seems like we should add this to Fabrikate definitions instead?

Reply:
Created #116